ASWF/gsoc_2021_README.md

```diff
@@ -0,0 +1,73 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+# OpenColorIO Google Summer of Code 2021
+
+OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production with an emphasis on visual effects and computer animation. OCIO provides a straightforward and consistent user experience across all supporting applications while allowing for sophisticated back-end configuration options suitable for high-end production usage. OCIO is compatible with the Academy Color Encoding Specification (ACES) and is LUT-format agnostic, supporting many popular formats.
+
+OpenColorIO version 1.0 was released in 2011, having been in development since 2003. OCIO v1 represented the culmination of years of production experience earned on such films as SpiderMan 2 (2004), Surf’s Up (2007), Cloudy with a Chance of Meatballs (2009), Alice in Wonderland (2010), and many more.  OpenColorIO v2 has been in development since 2017 and was feature complete as of SIGGRAPH 2020. After a stabilization and bug-fixing period, an official 2.0.0 release was made in January 2021! 
+
+While OCIO is a color management library (written in C++ with a Python API), its only knowledge of color science comes from its execution of the transforms defined in the OCIO configuration file, written in YAML. These transforms are either defined by the end user in a custom OCIO config or inherited from the publicly available configs.  By specifying your desired config file in the local environment all OCIO compatible applications and software libraries will be able to see your defined color transform “universe”. You can then direct the transformation of image data from one defined OCIO.ColorSpace to another, in addition to the other transforms documented elsewhere - the options are endless!
+
+## About the Academt Software Foundation
+
+The Academy Software Foundation (ASWF) is collaboration between the Linux
+Foundation and the Academy of Motion Picture Arts and Sciences, serving as
+an umbrella organization that is the home to several key open source
+software projects used by the film industry (primarily visual effects and
+animation production). The projects we manage are some of the most
+high-profile and widely used open source projects originating and used in
+film production studios. It's a great opportunity for students to get
+involved with projects that will be used professionally on films!
+
+The mission of the ASWF is to increase the quality and quantity of
+contributions to the content creation industry's open source software base;
+to provide a neutral forum to coordinate cross-project efforts; to provide a
+common build and test infrastructure; and to provide individuals and
+organizations a clear path to participation in advancing our open source
+ecosystem.
+
+Member projects are focused primarily on visual effects and animation
```

Comment:
I would delete this paragraph and replace with something like:

In 2020, the ASWF projects OpenEXR, OpenTimelineIO, and OpenCue mentored GSoC students.  This year, OpenColorIO is the ASWF project that looks forward to welcoming and mentoring GSoC students.